packages/algob/src/lib/tx.ts

```diff
@@ -247,3 +247,119 @@ export async function transferASALsig (
+  const txn = tx.makeApplicationNoOpTxn(
+    sender,
+    params,
+    appId,
+    appArgs,
+    accounts,
+    foreignApps,
+    foreignAssets,
+    note,
+    lease,
+    rekeyTo);
+
+  const txId = txn.txID().toString();
+  const signedTxn = txn.signTxn(sender.sk);
+  await deployer.algodClient.sendRawTransaction(signedTxn).do();
+  const transactionResponse = await deployer.waitForConfirmation(txId);
+
+  console.log("Called app-id:", appId);
+  if (transactionResponse['global-state-delta'] !== undefined) {
+    console.log("Global State updated:", transactionResponse['global-state-delta']);
+  }
+  if (transactionResponse['local-state-delta'] !== undefined) {
+    console.log("Local State updated:", transactionResponse['local-state-delta']);
+  }
+}
+
+/**
+ * Description: The user may discontinue use of the application by sending a close out transaction.
+ * This will remove the local state for this application from the user's account.
+ * @param deployer AlgobDeployer
+ * @param sender Account from which call needs to be made
+ * @param payFlags Transaction Flags
+ * @param appId ID of the application being configured or empty if creating
+ */
+export async function closeOutSSC (
+  deployer: AlgobDeployer,
+  sender: AccountSDK,
+  payFlags: TxParams,
+  appId: number): Promise<void> {
+  const params = await mkSuggestedParams(deployer.algodClient, payFlags);
+
+  const txn = tx.makeApplicationCloseOutTxn(sender, params, appId);
+
+  const txId = txn.txID().toString();
+  const signedTxn = txn.signTxn(sender.sk);
+  await deployer.algodClient.sendRawTransaction(signedTxn).do();
+  await deployer.waitForConfirmation(txId);
+}
+
+/**
+ * Description: The approval program defines the creator as the only account able to delete the application.
+ * This removes the global state, but does not impact any user's local state.
+ * @param deployer AlgobDeployer
+ * @param sender Account from which call needs to be made
+ * @param payFlags Transaction Flags
+ * @param appId ID of the application being configured or empty if creating
+ */
+export async function deleteTxnSSC (
+  deployer: AlgobDeployer,
+  sender: AccountSDK,
+  payFlags: TxParams,
+  appId: number): Promise<void> {
+  const params = await mkSuggestedParams(deployer.algodClient, payFlags);
+
+  const txn = tx.makeApplicationDeleteTxn(sender, params, appId);
+
+  const txId = txn.txID().toString();
+  const signedTxn = txn.signTxn(sender.sk);
+  await deployer.algodClient.sendRawTransaction(signedTxn).do();
+  await deployer.waitForConfirmation(txId);
+}
+
+/**
+ * Description: The user may clear the local state for an application at any time,
+ * even if the application was deleted by the creator.
+ * @param deployer AlgobDeployer
+ * @param sender Account from which call needs to be made
+ * @param payFlags Transaction Flags
+ * @param appId ID of the application being configured or empty if creating
+ */
+export async function clearStateSSC (
```

Comment:
How about renaming to `clearUserStateSSC` ... or in proper english it would be `clearSSCuserState`

Reply:
yes `clearSSCuserState` is better